game-core/src/main/java/games/strategy/engine/framework/startup/ui/ClientSetupPanel.java

```diff
@@ -63,7 +63,7 @@ private void internalPlayersChanged() {
       // clients only get to change bot settings
       disableable.clear();
     }
-    playerRows = new ArrayList<>();
+    playerRows.clear();
```

Comment:
Instead of reassigning a new ArrayList each time, we can just clear the current one and populate it again